pandas/core/internals/managers.py

```diff
@@ -663,7 +626,7 @@ def consolidate(self: T) -> T:
         if self.is_consolidated():
             return self
 
-        bm = type(self)(self.blocks, self.axes, self.refs, verify_integrity=False)
+        bm = type(self)(self.blocks, self.axes, verify_integrity=False)
```

Comment:
There isn't a call where this would matter right now. It's either coming from copy or from a function where we did a copy of the blocks anyway. At least I couldn't find any. Could add it though if you prefer.